ironfish/src/rpc/adapters/secureTcpAdapter.ts

```diff
@@ -1,37 +1,68 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import fs from 'fs'
 import net from 'net'
+import { pki } from 'node-forge'
 import tls from 'tls'
 import { createRootLogger, Logger } from '../../logger'
+import { ErrorUtils } from '../../utils'
 import { ApiNamespace } from '../routes'
 import { TcpAdapter } from './tcpAdapter'
 
 export class SecureTcpAdapter extends TcpAdapter {
-  readonly nodeKey: string
-  readonly nodeCert: string
+  readonly nodeKeyPath: string
+  readonly nodeCertPath: string
 
   constructor(
     host: string,
     port: number,
-    nodeKey: string,
-    nodeCert: string,
+    nodeKeyPath: string,
+    nodeCertPath: string,
     logger: Logger = createRootLogger(),
     namespaces: ApiNamespace[],
   ) {
     super(host, port, logger, namespaces)
-    this.nodeKey = nodeKey
-    this.nodeCert = nodeCert
+    this.nodeKeyPath = nodeKeyPath
+    this.nodeCertPath = nodeCertPath
   }
 
   protected createServer(): net.Server {
-    const options = {
-      host: this.host,
-      port: this.port,
-      key: this.nodeKey,
-      cert: this.nodeCert,
+    return tls.createServer(this.getTlsOptions(), (socket) => this.onClientConnection(socket))
+  }
+
+  protected getTlsOptions(): tls.TlsOptions {
+    try {
+      const nodeKey = fs.readFileSync(this.nodeKeyPath)
+      const nodeCert = fs.readFileSync(this.nodeCertPath)
+      return {
+        key: nodeKey,
+        cert: nodeCert,
+      }
+    } catch (e) {
+      if (ErrorUtils.isNoEntityError(e)) {
```

Comment:
I know it's common in Python to catch errors and perform logic based on that, but I think here it's probably better if we just check if the files exist on line 36 before we do this. You use `fs.access()` 
 like we do in `fileStore.ts` which is the new node way to check a file exists. Though I think we should just write a wrapper in FileSystem that adds `exists()`

https://github.com/iron-fish/ironfish/blob/master/ironfish/src/fileStores/fileStore.ts#L22-L25